api/src/main/java/org/pmiops/workbench/db/model/CohortAnnotationDefinition.java

```diff
@@ -78,6 +83,30 @@ public CohortAnnotationDefinition annotationType(AnnotationType annotationType)
         return this;
     }
 
+    @OneToMany(fetch = FetchType.EAGER, mappedBy = "cohortAnnotationDefinition", orphanRemoval = true, cascade = CascadeType.ALL)
+    public Set<CohortAnnotationEnumValue> getEnumValues() {
+        return enumValues;
+    }
+
+    public void setEnumValues(Set<CohortAnnotationEnumValue> enumValues) {
+        this.enumValues = enumValues;
+        if (enumValues != null) {
+            for (CohortAnnotationEnumValue enumValue : enumValues) {
+                enumValue.setCohortAnnotationDefinition(this);
+            }
+        }
+    }
+
+    public CohortAnnotationDefinition enumValues(Set<CohortAnnotationEnumValue> enumValues) {
+        this.enumValues = enumValues;
+        if (enumValues != null) {
+            for (CohortAnnotationEnumValue enumValue : this.enumValues) {
```

Comment:
Same as above? Shouldn't this just call setEnumValues()?

Reply:
Done.